### PR TITLE
dex: basic functionality

### PR DESCRIPTION
**Pull request by 0xalecks**
configures the dex-ui projects with basics:
- routing
- wagmi/wallet connection
- sdk injection
- /wells and /wells/:address pages functional

### CODE REVIEW

**Review by marshall-bean — CHANGES_REQUESTED** [review]
I think I figured out the issue with requiring modules. Maybe give it a try and it can be fixed in this PR. Otherwise, we can get this merged and then fix it separately.

File: projects/dex-ui/.eslintrc [hunk 1]
```diff
@@ -5,10 +5,9 @@
     }
   },
   "extends": [
-    "eslint:recommended",
     "plugin:react/recommended",
     "plugin:jsx-a11y/recommended",
-    "plugin:@typescript-eslint/recommended",
+    // "plugin:@typescript-eslint/recommended",
```
[the remark this answers is not in the document]
Comment (0xalecks) [reply projects/dex-ui/.eslintrc:10]: think we don't want it, but didn't want to remove it until we were sure

File: projects/dex-ui/src/wells/useWells.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  const [error, setError] = useState<Error>();
+
+  useEffect(() => {
+    let listen = true;
+    const fetchData = async () => {
+      setLoading(true);
+      setError(undefined);
+      const wells = await Promise.all(
+        WELL_ADDRESSES.map((address) => {
+          console.log(address);
```
Comment (marshall-bean) [inline projects/dex-ui/src/wells/useWells.tsx:19]: Errant console.log?

**Review by marshall-bean — CHANGES_REQUESTED** [review]

File: projects/dex-ui/package.json [hunk 3]
```diff
@@ -25,6 +25,7 @@
     "react-jazzicon": "1.0.4",
     "react-router-dom": "^6.4.4",
     "styled-components": "5.3.9",
+    "typescript": "4.8.4",
```
[the remark this answers is not in the document]
Comment (0xalecks) [reply projects/dex-ui/package.json:28]: i did this to match the monorepo version, thus removing the local typescript in the local node_modules folder

**Pull request merged**